### PR TITLE
IE 11 fixes for react-sortable-hoc

### PR DESCRIPTION
**Pull request by humiston**
Added window.pageYOffset and window.pageXOffset to updatePosition and animateNode to allow proper drag and drop animation for Internet Explorer 11

### CODE REVIEW

File: src/SortableContainer/index.js [hunk 1]
```diff
@@ -513,15 +513,15 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
...
       const offset = this.getOffset(e);
       const translate = {
         x: offset.x - this.initialOffset.x,
         y: offset.y - this.initialOffset.y,
       };
       // Adjust for window scroll
-      translate.y -= (window.scrollY - this.initialWindowScroll.top);
-      translate.x -= (window.scrollX - this.initialWindowScroll.left);
+      translate.y -= (window.scrollY - this.initialWindowScroll.top) || window.pageYOffset;
+      translate.x -= (window.scrollX - this.initialWindowScroll.left) || window.pageXOffset;
```
Comment (clauderic) [inline src/SortableContainer/index.js]: Looks to me like those two statements are not equivalent. 

The first one will take into account the diff between the current `scrollY` and `scrollX` values vs the initial scroll value

Comment (humiston) [inline src/SortableContainer/index.js]: I've updated this pr with the correct assignment of our initial scroll values so our diff is accurate when using IE 11.

**Comment by clauderic** [comment]
Just published `0.6.6` with this fix, thanks again for your contribution 😄

**Comment by mccambridge** [comment]
@clauderic @humiston I'm running storybook with this fix, and I still see the issue with IE11. Both in my code and in your Drag Handle storybook example. Items aren't draggable at all.

**Comment by timothyallan** [comment]
It's because IE11 doesn't support Array.From() which is being used in the code. I just ran into this myself.

**Pull request merged**